watchdog/files/watchdog.conf

```diff
@@ -0,0 +1,47 @@
+{%- from "watchdog/map.jinja" import server with context %}
+#ping                   = 172.31.14.1
```

Comment:
watchdog.conf 
would be worth from the beginning add pillar structure to support some of the conf optoins. (so the pillar is later properly formated).

For example file /var/log/messages should be possible to modify for centos..

Reply:
I would like to leave it for the future... Now I would like to get it merged and we can add additional features later.

Reply:
I don't see a reason why you can't do it now, but OK, do it in second pull request if you want to.